### DISCUSSION TITLE
.net6/8

### DISCUSSION THREAD

**Discussion by bc-tylertech** [SolrNet/SolrNet#620] [Q&A]
Are there any plans to add a .net6 or .net8 version of these nugets?

**Comment by mausch** [comment]
I don't think there's any need since the packages are .net standard 2.0?
Do you have anything specific in mind that would require publishing version-specific packages?

**Reply by bc-tylertech** [reply]
Just keeping up with the latest apis and efficiencies, but if the Solr codebase isn't using any of those, then maybe there's no need?  I mostly wanted to make sure this lib was being actively developed, and it looks like that's the case!  Thanks!

**Reply by mausch** [reply]
> latest apis and efficiencies

If you're thinking of anything specific let me know.
